include/lsst/jointcal/MeasuredStar.h

```diff
@@ -99,7 +99,8 @@ class MeasuredStar : public BaseStar {
     double _instFluxErr;
 
     const CcdImage *_ccdImage;
-    std::shared_ptr<const FittedStar> _fittedStar;
+    // Note: _fittedStar is not const, but we should not be modifying it within a measuredStar.
+    std::shared_ptr<FittedStar> _fittedStar;
```

Comment:
I'd word this as "`MeasuredStar` won't touch it". It's *your* class now, *you* should control what it does or doesn't do.